src/DeVinci_frontend/helpers/localStorage.js

```diff
@@ -3,8 +3,7 @@ import {
   saveChatsUserSelection
 } from "../store";
 
-let storeState;
-store.subscribe((value) => storeState = value);
```

Comment:
can we refactor this the way I did? because of the error below:

localStorage.js:12 Uncaught ReferenceError: Cannot access 'store' before initialization
at localStorage.js:12:1

Reply:
this doesn't work; the file also shows errors (as backendActor is not a field on store), so we need to revert this.

We don't need the function initializeStores, calling it in App causes the error